### ISSUE TITLE
Implement a plugin for date and time picking

### ISSUE THREAD

**Issue by tgpetrov** [NativeScript/nsplugins-internal#177]

**Comment by tgpetrov** [comment]
The plugin is here: https://github.com/nativeScript/nativescript-datetimepicker. It provides two fields - `DatePickerField` and `TimePickerField` - both are NativeScript Views that show selected date or time and allow picking another after being tapped. There is also a `DateTimePicker` class which provides static methods `pickDate` and `pickTime` that can be called to show the same dialog picker as the fields.

**Comment by DimitarTodorov** [comment]
Issues:
    - [ ] Fix scripts in package.json
    - [ ] Issue with keyboard when tap on CSS field on app Resume
    - [ ] Demo-vue launch image asset safe area

**Comment by tgpetrov** [comment]
@DimitarTodorov 

I have fixed the launch image for demo-vue.

I managed to reproduce the issue with the Keyboard also in tns-core-modules' `TextField` in [this playground](https://play.nativescript.org/?template=play-ng&id=JPz7Zm). It happens if you quickly tap on any `TextField` with `editable="false"` (as are all `DatePickerField`-s and `TimePickerField`-s) while the app is resuming. I logged the issue in the NativeScript repo.
